### PR TITLE
Update WanderlustDate toString method

### PR DESCRIPTION
**Pull request by jeannetoh99**

### CODE REVIEW

File: src/main/java/seedu/address/model/commons/WanderlustDate.java [hunk 1]
```diff
@@ -55,7 +56,7 @@ public LocalDate getValue() {
 
     @Override
     public String toString() {
-        return value.toString();
+        return value.format(DateTimeFormatter.ofPattern("MMM dd yyyy"));
```
Comment (underthehai) [inline src/main/java/seedu/address/model/commons/WanderlustDate.java:59]: Is it supposed to be MM instead of 3 'M's?

Comment (underthehai) [reply src/main/java/seedu/address/model/commons/WanderlustDate.java:59]: ohh its abbreviated form!

**Review by underthehai — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/commons/WanderlustDate.java (hunk 1 above)
Comment (jeannetoh99) [reply src/main/java/seedu/address/model/commons/WanderlustDate.java:59]: yeap! It's meant to be Jan, Feb, etc.

**Pull request merged**